lua/astrocommunity/pack/cpp/init.lua

```diff
@@ -46,6 +46,12 @@ return {
                 if assert(vim.lsp.get_client_by_id(args.data.client_id)).name == "clangd" then
                   require "clangd_extensions"
                   vim.api.nvim_del_augroup_by_name "clangd_extensions"
+                  vim.keymap.set(
```

Comment:
Please use astrocore to set keymaps

Reply:
https://github.com/AstroNvim/astrocore/blob/main/doc/api.md#set_mappings
Did you mean this api ?